chemicaltoolbox/openduck/openduck_run_smd.xml

```diff
@@ -0,0 +1,105 @@
+<tool id="openduck_run_smd" name="Run OpenDUck" version="@VERSION@">
+    <description>steered molecular dynamics runs</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        ## to run on GPU set OPENDUCK_GPU_PARAM to something like "--gpu-id 1"
+        cp '$ligand' ./ligand.sdf &&
+        
+        duck_prepare_sys
+            -p '$protein'
+            -l ./ligand.sdf
+            -c '$chunk'
+            ## -i is interaction, e.g. A_ASP_156_OD2
+            -i '${ia.chain_sel}'_'${ia.res_sel}'_'${ia.resid_sel}'_'${ia.atom_sel}' 
+            `echo "\${OPENDUCK_GPU_PARAM}"` &&
```

Comment:
```suggestion
            \${OPENDUCK_GPU_PARAM} &&
```

Reply:
I hope this works.

Reply:
Yes, it's pretty nasty. Otherwise python gives something like `error: unrecognized arguments:''`

Reply:
will try your version out